### PR TITLE
mgr/dashboard: revise monitoring documentation

### PR DESCRIPTION
**Pull request by p-se**
This PR contains changes from PR https://github.com/ceph/ceph/pull/34902 and shall not be merged before its predecessor.

## Checklist
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
 
-Grafana and Prometheus are likely going to be bundled and installed by some
-orchestration tools along Ceph in the near future, but currently, you will have
-to install and configure both manually. After you have installed Prometheus and
-Grafana on your preferred hosts, proceed with the following steps.
+Installation and Configuration using cephadm
+""""""""""""""""""""""""""""""""""""""""""""
 
-1. Enable the Ceph Exporter which comes as Ceph Manager module by running::
+Grafana and Prometheus can be installed using :ref:`cephadm`. It will
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:408]: What do you mean by "it" here? The Dashboard?

File: doc/mgr/dashboard.rst [hunk 2]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
 
-5. Add the Dashboards to Grafana:
+    .. note::
 
-  Dashboards can be added to Grafana by importing dashboard jsons.
-  Following command can be used for downloading json files::
+      Please note that in the aforementioned example, Prometheus is configured
+      to scrape data from itself (port 9090), the Ceph manager module
+      `prometheus` (port 9283), which exports Ceph internal data and the Node
+      exporter (port 9100), which provides metrics of a machine.
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:448]: Shouldn't this be plural?
```suggestion
      exporters (port 9100), which provide metrics of all nodes.
```

File: doc/mgr/dashboard.rst [hunk 3]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
+      
+      Depending on your configuration, you may need to change the hostname in
+      this configuration or add additional configuration entries for the Node
+      exporter.  It is unlikely that you will need to change the provided
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:452]:
```suggestion
      exporter. It is unlikely that you will need to change the provided
```

File: doc/mgr/dashboard.rst [hunk 4]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
+      ports.
+      
+      Moreover, you don't *need* to have more than one target for Ceph specific
+      data, provided by the `prometheus` mgr module.  But it is recommended to
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:456]:
```suggestion
      data, provided by the `prometheus` mgr module. But it is recommended to
```

File: doc/mgr/dashboard.rst [hunk 5]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
 
-  For Example, for ceph-cluster overview you can use::
+      grafana-cli plugins install vonage-status-panel
+      grafana-cli plugins install grafana-piechart-panel
 
-    wget https://raw.githubusercontent.com/ceph/ceph/master/monitoring/grafana/dashboards/ceph-cluster.json
+#.  Add the Dashboards to Grafana:
 
-6. Configure Grafana in `/etc/grafana/grafana.ini` to adapt anonymous mode::
+    Dashboards can be added to Grafana by importing dashboard jsons.
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:472]:
```suggestion
    Dashboards can be added to Grafana by importing dashboard JSON files.
```

File: doc/mgr/dashboard.rst [hunk 6]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
+    Following command can be used for downloading json files::
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:473]:
```suggestion
    Use the following command to download the JSON files::
```

File: doc/mgr/dashboard.rst [hunk 7]
```diff
@@ -402,71 +402,107 @@ separate application that provides machine metrics.
...
-  set to ``true`` for the Grafana integration in Ceph Dashboard to work, as its
-  default is ``false``.
+    You can find all the dashboard jsons `here <https://github.com/ceph/ceph/tree/
+    master/monitoring/grafana/dashboards>`_ .
+
+    For Example, for ceph-cluster overview you can use::
+
+      wget https://raw.githubusercontent.com/ceph/ceph/master/monitoring/grafana/dashboards/ceph-cluster.json
+
+#.  Configure Grafana in `/etc/grafana/grafana.ini` to adapt anonymous mode
```
Comment (LenzGr) [inline doc/mgr/dashboard.rst:484]: I think reStructuredText uses double backticks to mark monospaced text:
```suggestion
#.  Configure Grafana in ``/etc/grafana/grafana.ini`` to adapt anonymous mode
```

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by LenzGr** [comment]
Rendered pages for review:

- https://docs.ceph.com/ceph-prs/34928/cephadm/monitoring/
- https://docs.ceph.com/ceph-prs/34928/mgr/dashboard/#installation-and-configuration-using-cephadm

File: doc/mgr/dashboard.rst (hunk 1 above)
Comment (p-se) [reply doc/mgr/dashboard.rst:408]: Good spotting :sweat_smile: Both components will automatically be configured by cephadm. I'll adapt it to `they`.

File: doc/mgr/dashboard.rst (hunk 2 above)
Comment (p-se) [reply doc/mgr/dashboard.rst:448]: The note relates to the example configuration where only one node exporter is configured, thus I think, singular should be correct.

**Comment by p-se** [comment]
jenkins test make check

**Comment by p-se** [comment]
jenkins test make check

**Comment by LenzGr** [comment]
jenkins render docs

**Comment by LenzGr** [comment]
jenkins run make check

**Comment by LenzGr** [comment]
jenkins test make check

**Comment by LenzGr** [comment]
jenkins test make check

**Comment by p-se** [comment]
jenkins test make check

**Pull request merged**